tests/unit/products.t

```diff
@@ -234,4 +234,21 @@ foreach my $rem_field (@$fields_to_remove) {
 }
 is($product_ref->{name}, "test_prod");
 
+# Test that NOVA and estimated % of fruits and vegetables are ignored when determining if the nutrients are completed.
+$product_ref->{nutriments} = {
+	fruits-vegetables-nuts-estimate-from-ingredients_100g => 0,
```

Comment:
Ah in fact this makes the tests fail:

```
Bareword "fruits" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "vegetables" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "nuts" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "estimate" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "from" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "fruits" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "vegetables" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "nuts" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "estimate" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "from" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "nova" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "nova" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Bareword "nova" not allowed while "strict subs" in use at tests/unit/products.t line 239.
Execution of tests/unit/products.t aborted due to compilation errors.
tests/unit/products.t ......................... 
```

It's because keys that have a dash need to be quoted:
"fruits-vegetables-nuts-estimate-from-ingredients_100g" => 0,

Reply:
i had trouble the test pass @stephanegigandet actually i choose to give it some time.
hope it passes now